dds_web/templates/mail/project_release.html

```diff
@@ -28,6 +30,10 @@
         The DDS CLI command <i>dds data get -p {{project_id}} -a</i> can be used to download all the files in this project to your current directory. </p>
       <p style=" font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-top: 15px;">
         Your access to this project will expire on {{deadline}}</p>
+      <p style="font-family: arial, sans-serif; font-size: 10px; font-weight: normal; margin: 0; Margin-top: 10px;">
+          What is the DDS? The DDS is a system for SciLifeLab infrastructures to deliver data to researchers in a fast, secure and simple
```

Comment:
Just to emphasize it a bit more
```suggestion
          <b>What is the DDS?</b> The DDS is a system for SciLifeLab infrastructures to deliver data to researchers in a fast, secure and simple
```